DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "CrowdStrikeFalconEndpointProtection",
+    "title": "CrowdStrike Falcon Endpoint Protection",
+    "publisher": "CrowdStrike",
+    "descriptionMarkdown": "The [CrowdStrike Falcon Endpoint Protection](https://www.crowdstrike.com/endpoint-security-products/) connector allows you to easily connect your CrowdStrike Falcon Event Stream with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization's endpoints and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CrowdStrikeFalconEventStream** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-crowdstrikefalcon-parser)",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "CrowdStrikeFalconEventStream",
+        "baseQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"CrowdStrike\" and DeviceProduct == \"FalconHost\""
```

Comment:
There is a special character next to "CommonSecurityLog" (in 3 places), Can you please remove that. Please check the attached screenshot that gives the placing.
![CrowdStrike-SpecialCharacters](https://user-images.githubusercontent.com/73506651/103897803-1055f380-511a-11eb-81b7-be76785f9f88.PNG)

Reply:
@v-jayakal  I wasnt able to see if same. I deleted the space and added it, let me know if that fixed it

Reply:
updated made